### PR TITLE
Update community-plugin-marketplace.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Removed double line breaks, changes high-availability to High Availability.

### CODE REVIEW

**Review by hanzei — APPROVED** [review]

File: site/content/extend/plugins/community-plugin-marketplace.md [hunk 1]
```diff
@@ -4,45 +4,53 @@ subsection: Plugins (Beta)
...
 
+**Documentation requirements (checked by a Technical Writer)**
+
+1. The plugin must include a `README.md` file with:
+    * Requirements
+    * Installation steps
+    * Configuration steps
+    * Usage
+    * Link or email address for help/support
+2. The `plugin.json` file should include formatting consistent with the System Console UI. For assistance with this, you can visit the [Documentation channel](https://community.mattermost.com/core/channels/documentation) for assistance.
```
Comment (jasonblais) [inline site/content/extend/plugins/community-plugin-marketplace.md:47]:
```suggestion
2. The `plugin.json` file should include formatting consistent with the System Console UI. You can join the [Documentation channel](https://community.mattermost.com/core/channels/documentation) for assistance.
```

**Review by jasonblais — APPROVED** [review]
One non-blocking proposal

File: site/content/extend/plugins/community-plugin-marketplace.md (hunk 1 above)
Comment (hanzei) [inline site/content/extend/plugins/community-plugin-marketplace.md:47]: Are there any docs about the style guidelines that we can link?

File: site/content/extend/plugins/community-plugin-marketplace.md [hunk 2]
```diff
@@ -4,45 +4,53 @@ subsection: Plugins (Beta)
...
 2. The plugin must work on all Mattermost versions greater than or equal to the `min_server_version`.
 
+**Documentation requirements (checked by a Technical Writer)**
+
+1. The plugin must include a `README.md` file with:
+    * Requirements
+    * Installation steps
+    * Configuration steps
+    * Usage
+    * Link or email address for help/support
```
Comment (hanzei) [inline site/content/extend/plugins/community-plugin-marketplace.md:46]: With these items in place does that mean item 4. at "Product requirements" is not longer needed? Should we combine those?

Comment (hanzei) [inline site/content/extend/plugins/community-plugin-marketplace.md:46]: Can the support link just be the github issues page? If so, what is the difference to item 3 on "Product requirements"?

**Review by hanzei — CHANGES_REQUESTED** [review]
Heads up that the changes here will need to get propagated to https://github.com/mattermost/mattermost-marketplace/blob/master/.github/ISSUE_TEMPLATE/add_plugin.md. @justinegeffen is that something you can do after this PR is merged?

File: site/content/extend/plugins/community-plugin-marketplace.md (hunk 2 above)
Comment (justinegeffen) [reply site/content/extend/plugins/community-plugin-marketplace.md:46]: Good catch. I'll combine them.

File: site/content/extend/plugins/community-plugin-marketplace.md [hunk 3]
```diff
@@ -5,45 +5,53 @@ description: "Once your plugin has reached a certain level of quality, you might
...
-2. The source code is available in a public git repository.
+2. The source code is available in a public Git repository.
 3. There is a public issue or bug tracker for the plugin, which is linked in the plugin documentation and linked via `support_url` in the manifest.
 4. The plugin provides detailed usage documentation with at least one screenshot of the plugin in action, list of features, and a development guide. This is typically a `README` file or a landing page on the web. The link to the documentation is set as `homepage_url` in the manifest. A great example is the [`README` of the GitHub plugin](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md).
 5. For the current release and future releases, a changelog must be published. The link to the current release notes has to be recorded in the `release_notes_url` property of the `plugin.json` manifest. For example GitHub releases can be used to publish the changelog.
 6. The plugin has to be out of Beta and be released with at least v1.0.0.
 7. All configuration is accessible via the Mattermost UI.
```
Comment (justinegeffen) [inline site/content/extend/plugins/community-plugin-marketplace.md:22]:
```suggestion
7. All configuration is accessible via the Mattermost interface.
```

File: site/content/extend/plugins/community-plugin-marketplace.md [hunk 4]
```diff
@@ -5,45 +5,53 @@ description: "Once your plugin has reached a certain level of quality, you might
...
 8. The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Plugin Marketplace. It should follow [the documentation's suggested naming convention](https://developers.mattermost.com/extend/plugins/manifest-reference/#id).
```
Comment (justinegeffen) [inline site/content/extend/plugins/community-plugin-marketplace.md:23]:
```suggestion
8. The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Marketplace. It should follow [the documentation's suggested naming convention](https://developers.mattermost.com/extend/plugins/manifest-reference/#id).
```

**Pull request closed without merging**